src/appleseed.python/bindframe.cpp

```diff
@@ -142,7 +142,6 @@ void bind_frame()
         .def("image", &Frame::image, bpy::return_value_policy<bpy::reference_existing_object>())
         .def("aov_images", &Frame::aov_images, bpy::return_value_policy<bpy::reference_existing_object>())
 
```

Comment:
We're not exposing `Frame::clear_main_and_aov_images()`?

Reply:
There's no use for it in python I think.
I plan to clean the python bindings for the 1.9 release. There are a number of classes and methods that shouldn't be exposed. This method was one of them.